### DISCUSSION TITLE
Cannot embed trivial MathJax

### DISCUSSION THREAD

**Discussion by hirrolot** [jgm/pandoc#10061] [Q&A] [answered]
I cannot use MathJax with Pandoc:

```
$ pandoc --version
pandoc 3.3
$ cat test.md
Here is an inline equation: \( E = mc^2 \).
$ pandoc test.md --mathjax -o out.html
$ cat out.html
<p>Here is an inline equation: ( E = mc^2 ).</p>
```

The output HTML renders as text, not as a MathJax formula. What am I doing wrong?

My OS is Ubuntu 22.04.

**Comment by hirrolot** [comment, marked as answer]
So I was misreading the documentation. The correct syntax is:

```
Here is an inline equation: $E = mc^2$.
```

**Reply by tarleb** [reply]
Alternatively, one can also switch on support for the `\( ... \)` syntax by enabling the `tex_math_single_backslash` extensions:

    pandoc --from=markdown+tex_math_single_backslash
